kfs-core/src/main/resources/edu/arizona/kfs/fp/document/workflow/AuxiliaryVoucherDocument.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='UTF-8'?> 
```

Comment:
We are putting all edu.arizona workflow files here:
kfs-core/src/main/resources/edu/arizona/kfs/workflow
Please move.